webpack.grav.config.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Grav-specific webpack config
+ * This is merged over top of webpack.shared.config.js
+ */
+const path = require('path');
+const merge = require('webpack-merge');
+const shared = require('./webpack.shared.config');
```

Comment:
I'd like to refactor these webpack configs to move the custom imports (`shared`) to the bottom with a space between custom imports and library imports.